### PR TITLE
use post-install.txt instead of manifest.json

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| License       | MIT

https://github.com/symfony/recipes-contrib/pull/565

cc @makasim

### CODE REVIEW

File: enqueue/amqp-bunny/0.8/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<bg=blue;fg=white>               </>
+<bg=blue;fg=white> New transport </>
+<bg=blue;fg=white>               </>
+
+  * the DSN used in ENQUEUE_DSN should start with <fg=green>amqp+bunny://</>.
```
Comment (OskarStark) [inline enqueue/amqp-bunny/0.8/post-install.txt]: please give me some feedback on the post-install.txt

**Comment by OskarStark** [comment]
Hmm, shall I add empty manifest.json @Nyholm ?

**Comment by Nyholm** [comment]
Thank you. 
I see that you removed the manifest file for amqp-bunny:0.8. Same for other packages. You also have to remove the package for amqp-bunny:0.7 or Flex will automatically use that recipe instead of the 0.8 one.

**Comment by Nyholm** [comment]
> Hmm, shall I add empty manifest.json @Nyholm ?

That seams even weirder... Can I install the bundle 0.9 with amqp-bunny:0.7? Probably, right? If so, I would remove all recipes for adapters.

**Comment by OskarStark** [comment]
> I see that you removed the manifest file for amqp-bunny:0.8. Same for other packages. You also have to remove the package for amqp-bunny:0.7 or Flex will automatically use that recipe instead of the 0.8 one.

good to know 👍 

> That seams even weirder... Can I install the bundle 0.9 with amqp-bunny:0.7? Probably, right? If so, I would remove all recipes for adapters.

Hmm yes, but in this case everybody needs to go to the docs and check how the DSN should start.
would be great if one could only add an `info.txt` for such information.

**Comment by Nyholm** [comment]
Hm.. Yeah, a recipe is never complete without a `manifest.json`. So I would assume that it would ignore the "recipe" in `enqueue/amqp-bunny/0.8`.

> but in this case everybody needs to go to the docs and check how the DSN should start.

True, but that is probably where you find out about the transport anyways, right? Ie https://github.com/php-enqueue/enqueue-dev/blob/master/docs/transport/amqp_bunny.md

**Comment by OskarStark** [comment]
Yes, maybe you are right and we should remove the recipes for the transports

**Comment by Nyholm** [comment]
Sorry Oskar, I missed this PR when merging https://github.com/symfony/recipes-contrib/pull/596#pullrequestreview-190597289.

I'll close this. Please let me know if there is still something in here that should be merged.

**Comment by OskarStark** [comment]
No everything looks fine 👌🏻

**Pull request closed without merging**